package.json

```diff
@@ -24,6 +24,18 @@
     "style-loader": "^3.3.1",
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.1",
-    "webpack-dev-server": "^4.11.1"
+    "webpack-dev-server": "^4.11.1",
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "hint": "^7.1.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.2.1"
```

Comment:
I suggest you fix this dependency since it runs the code with errors. The errors are displayed on the browser as well as the vscode terminal.
<img width="1024" alt="Screenshot 2023-01-21 at 07 36 13" src="https://user-images.githubusercontent.com/48546992/213843742-bc34896c-6903-478a-b2f3-62aff34e5acd.png">
Always ensure that before asking for a review your code is working as expected.

Reply:
Hello @Nyame-Wolf 
Thanks for your review, However, I am not able to find neither any browser errors or console errors, I have attached a small video to show what my app looks like on my local computer. Thanks:)

https://user-images.githubusercontent.com/55967946/213844244-ed607d1a-4ac6-4dcf-ad75-c2e8bfd4707e.mp4

Reply:
@Nyame-Wolf 
Please suggest to me what to do now! Thanks